modules/rostests/apitests/shlwapi/SHPropertyBag.cpp

```diff
@@ -64,13 +89,36 @@ class CDummyPropertyBag : public IPropertyBag
             {
                 ok_wstr(pszPropName, s_pszPropNames[i]);
                 s_pszPropNames[i] = NULL;
+
                 if (lstrcmpiW(pszPropName, L"RECTL2.top") == 0)
                     return E_FAIL;
 
+                if (lstrcmpiW(pszPropName, L"GUID1") == 0)
+                {
+                    V_VT(pvari) = (VT_UI1 | VT_ARRAY);
+                    V_ARRAY(pvari) = CreateByteArray(&IID_IShellLink, sizeof(IID));
+                    return S_OK;
+                }
+
+                if (lstrcmpiW(pszPropName, L"GUID2") == 0)
+                {
+                    V_VT(pvari) = VT_BSTR;
+                    V_BSTR(pvari) =
+                        SysAllocString(L"{00000000-0000-0000-C000-000000000046}"); // IID_IUnknown
```

Comment:
why this value specifically?

Reply:
Fixed with using `StringFromGUID2`.

Reply:
what I meant is, why using this GUID and not another one?

Reply:
Another GUID is also okay if the tests could be performed.

Reply:
Another GUID is also okay if the tests could be performed.

Reply:
Another GUID is also okay if the tests could be performed.

Reply:
Another GUID is also okay if the tests could be performed.

Reply:
Another GUID is also okay if the tests could be performed.

Reply:
Another GUID is also okay if the tests could be performed.

Reply:
Using another GUID is also okay.

Reply:
I intended to choose famous and available one.

Reply:
Is there something wrong with using this guid?

Reply:
I suppose nothing is wrong. I was just asking in order to know whether any GUID value was as good as any other for the present test, or whether that value was chosen because e.g. this was a value that was expected from testing on Windows.